gulpfile.js

```diff
@@ -138,6 +138,8 @@ declareExtension('amp-viewer-integration', '0.1', {
 declareExtension('amp-video', '0.1', false);
 declareExtension('amp-youtube', '0.1', false);
 declareExtensionVersionAlias(
+    'amp-sidebar', '0.1', /* lastestVersion */ '1.0', /* hasCss */ true);
```

Comment:
this is not ok. It basic copy built 1.0 version to 0.1 version. Should only be added after 0.1 version is fully deprecated.